File: Frontend/Logic/B20/Frontend/Windows/Context/Impl.cs
using B20.Architecture.Contexts.Api;
using B20.Frontend.Windows.Api;
using B20.Frontend.Windows.Impl;

namespace B20.Frontend.Windows.Context
{
    public class WindowsImpl: ContextModule
    {
        public void Apply(ContextBuilder builder)
        {
            builder.SetImpl<WindowManager, WindowManagerLogic>();
        }
    }
}

File: Backend/lib/src/main/kotlin/com/github/bratek20/ostium/gamesmanagement/context/Impl.kt
package com.github.bratek20.ostium.gamesmanagement.context

import com.github.bratek20.architecture.context.api.ContextBuilder
import com.github.bratek20.architecture.context.api.ContextModule

import com.github.bratek20.ostium.gamesmanagement.api.*
import com.github.bratek20.ostium.gamesmanagement.impl.*

class GamesManagementImpl: ContextModule {
    override fun apply(builder: ContextBuilder) {
        builder
            .setImpl(GamesManagementApi::class.java, GamesManagementApiLogic::class.java)
    }
}